src/pages/createAppointment/selectPost.tsx

```diff
@@ -0,0 +1,75 @@
+import { useContext, useEffect, useState } from "react";
+import { NextRouter, useRouter } from "next/router";
+import { userContext } from "supabase/user_context";
+import { useSupabaseClient } from "@supabase/auth-helpers-react";
+import { Database } from "supabase/db_types";
+import { Box, Typography } from "@mui/material";
+import Loading from "@/components/public/Loading";
+import Navbar from "@/components/public/Navbar";
+import { Post } from "@/types/Post";
+import { PagePaths } from "enum/pages";
+import { GetCurrentUserPosts } from "@/services/Posts";
+import { Grid } from "@mui/material";
+import EachPostCard from "@/components/createAppointment/EachPostCard";
+
+const MainLayout = {
+  margin: "1vh 0 0 0",
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+  width: "100%",
+  flexDirection: "row",
+  padding: "30px",
+};
+
+export default function ShowAppointment() {
+  const router: NextRouter = useRouter();
+  const userStatus = useContext(userContext);
+  const supabaseClient = useSupabaseClient<Database>();
+  const [posts, setPosts] = useState<Post[] | null>(null);
+
+  function handleClickCard(item: Post) {
+    router.push(PagePaths.createAppointment + "/" + item?.postId);
+  }
+
+  useEffect(() => {
+    if (!userStatus.user) return;
+    GetCurrentUserPosts(userStatus.user, supabaseClient)
+      .then((p) => {
+        setPosts(p);
+      })
+      .catch((err) => {
+        console.log(err);
+        return;
+      });
+  }, [userStatus.user, supabaseClient]);
+
+  if (userStatus.isLoading) return <Loading />;
+  if (!userStatus.user) {
+    router.push(PagePaths.login);
+    return;
+  }
+  if (!posts) return <Loading />;
+  return (
+    <>
+      <Navbar />
+      {/*  Title */}
+      <Box sx={MainLayout}>
+        <Typography variant="h1" sx={{ fontWeight: "700", fontSize: "40px" }}>
+          Select Post To Create Appointment
+        </Typography>
+      </Box>
+      {/*  EachPostCard */}
```

Comment:
Didn't the date time format should be `dd / mm / yyyy hh.mm [AM/PM]`
<img width="468" alt="image" src="https://user-images.githubusercontent.com/62459518/224495307-80c5061f-85b8-45ff-b034-d51b3f4617cc.png">

Reply:
Actually,Yes.
But I change into this for easy reading.(I think)

Reply:
@Palm394 what do you think about this? and if this place has changed the format, do every place of this field should change into this too?

Reply:
Our date time picker use `dd / mm / yyyy hh.mm [AM/PM]`. So We should choose this format for consistency. (However, I just notice that normal people play boardgame in this system across the day? lol)

Reply:
@tumrabert Your idea is not bad.It follows material design v2 guideline. If you told designer early, this idea might be accepted.

Reply:
Done